app/services/foreman_ansible/playbooks_importer.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+module ForemanAnsible
+  # Imports playbooks from smart proxy
+  class PlaybooksImporter
+    include ::ForemanAnsible::ProxyAPI
+
+    def initialize(proxy = nil)
+      @ansible_proxy = proxy
+    end
+
+    def import_playbook
+      import_playbooks_content playbooks if @ansible_proxy.present?
+    end
+
+    def import_playbooks_content(playbooks)
+      playbooks.map | playbook | import_parsed(playbook)
```

Comment:
Should this be `playbooks.map { |playbook| import_parsed playbook }` ?

Also, `import_parsed` is not yet implemented.

Reply:
I am supposed to use import_parsed from JobTemplate or am I wrong? 
https://github.com/theforeman/foreman_remote_execution/blob/fe4613c6f2e7a0f1d184e51b3b5325115cdedb02/app/models/job_template.rb#L72

Reply:
Using existing code is good, so use it if fits your needs.